syntax = "proto3";

option go_package = "pb/game";

import "common/tip.proto";

message NodeInfo
{
  uint32 node_id = 1;
  uint32 node_type = 2;
  uint64 launch_time = 3;
  uint32 game_node_type = 4;
}

message IpPort
{
  string ip = 1;
  uint32 port = 2;
}

message TipMessage
{
  Tip tips = 1;
}
